easybuild/framework/easyconfig/default.py

```diff
@@ -134,6 +134,7 @@
     'keepsymlinks': [False, ('Boolean to determine whether symlinks are to be kept during copying '
                              'or if the content of the files pointed to should be copied'),
                      FILEMANAGEMENT],
+    'umask': ['', "Set custom umask in octal format", FILEMANAGEMENT],
```

Comment:
@pescobar default value should be `None`, not `''` (see the `umask is not None`) check